runtime/doc/options.txt

```diff
@@ -2950,6 +2950,20 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Join the current window in the group of windows that shows differences
 	between files.  See |vimdiff|.
 
+					    *'dia'* *'diffanchors'* *E1548*
+'diffanchors' 'dia'	string	(default "")
+			global or local to buffer |global-local|
+	List of {address} in each buffer, separated by commas, that are
```

Comment:
What would be the meaning of the anchor if address is a current line `.` or the whole buffer `%`?

Reply:
I guess for the current line it would just anchor it and then after `:diffupdate` I should be able to see results.

Reply:
and the whole buffer probably wouldn't make sense to anchor.

Reply:
> and the whole buffer probably wouldn't make sense to anchor.

It could make sense to anchor the whole file sometimes, if set using buffer-local option. So if you set buffer 1's anchor to `1,$+1`, and buffer 2 to say `'a,'b`, that means everything in buffer 1 will be diff'ed against only the part you specified in buffer 2. There *is* some use.

Reply:
> `%` would not work (setting it would trigger an error) because it's not a valid {address}

It is specified as equal to `1,$` but still within `:h {address}` though.

Reply:
> It is specified as equal to 1,$ but still within :h {address} though

Hm, I guess you are right. I would probably just specify in the 'diffanchors' docs that this is not supported than adding support for it, since it seems pretty low value for this feature.